packages/manager/apps/pci-workflow/src/pages/list/List.page.tsx

```diff
@@ -278,13 +280,33 @@ export default function ListingPage() {
           </OsdsPopover>
         </div>
       </div>
+      <div className="flex mt-4">
+        {searchQueries.map((query, index) => (
```

Comment:
can't you use addFilter with key = 'search' to avoid redefining a list of chips ?